.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install lint-staged
```

Comment:
husky v5 needs this instead - see https://typicode.github.io/husky/#/?id=migrate-from-v4-to-v5